cli/prompt_secret.ts

```diff
@@ -48,10 +48,6 @@ export function promptSecret(
 ): string | null {
   const { mask = "*", clear } = options ?? {};
 
-  if (!input.isTerminal()) {
-    return null;
-  }
-
```

Comment:
I don't think this should be removed. This is for avoid prompting in non terminal

Reply:
You can stub `isTerminal` to avoid errors in CI